Sources/MapboxNavigation/NavigationMapView.swift

```diff
@@ -268,6 +302,11 @@ open class NavigationMapView: UIView {
             self.updateUserCourseView(location)
         }
         
+        mapView.mapboxMap.onNext(.cameraChanged) { [weak self] _ in
```

Comment:
Maybe `onEvery` is more appropriate here?

Reply:
Yes, you're right. The location provider could be override and changed after the mapView set up. The `onEvery ` could make sure that the `passiveLocationUsed` is keeping on track with the change.